### PR TITLE
'support  application/octet-stream'

### PR DESCRIPTION
**Pull request by cheneywang**
support  application/octet-stream

### CODE REVIEW

**Comment by cheneywang** [comment]
why

File: request.js [hunk 1]
```diff
@@ -1176,12 +1176,14 @@ Request.prototype.qs = function (q, clobber) {
 Request.prototype.form = function (form) {
   var self = this
   if (form) {
-    if (!/^application\/x-www-form-urlencoded\b/.test(self.getHeader('content-type'))) {
-      self.setHeader('content-type', 'application/x-www-form-urlencoded')
+    //Support application/octet-stream
```
Comment (Huachao) [inline request.js]: add a space between double slash and 'S'

**Comment by FredKSchott** [comment]
I am happy to review after broken tests are fixed

**Comment by FredKSchott** [comment]
Closing as abandoned. Please reopen a new PR with passing tests if you'd still like to get this merged.

**Pull request closed without merging**